examples/problem/rbf.c

```diff
@@ -0,0 +1,66 @@
+/*! @copyright (c) 2020 King Abdullah University of Science and
+ *                      Technology (KAUST). All rights reserved.
+ *
+ * STARS-H is a software package, provided by King Abdullah
+ *             University of Science and Technology (KAUST)
+ *
+ * @file examples/problem/rbf.c
+ * @version 0.1.0
+ * @auther Rabab Alomairy
+ * @author Aleksandr Mikhalev
+ * @date 2020-05-17
+ * */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <starsh.h>
+#include <starsh-rbf.h>
+
+int main(int argc, char **argv)
+{
+    int problem_ndim = 3; // problem dimension
+    int kernel_type = 0;
```

Comment:
Since you have only single kernel for your problem, `kernel_type` can be removed entirely from examples and sources. However, if you plan to add new kernels, this comment is to be ignored

Reply:
we have many kernels for RBF. This could be 0, 1, 2, 3, 4, 5, 6, 7, and 8. Each one could be any of provided basis functoin